packages/php/README.md

```diff
@@ -1,48 +1,31 @@
+</p>
 
-## Installation
+With [ReadMe's Metrics API](https://readme.com/metrics) your team can get deep insights into your API's usage. If you're a developer it's a few small steps to send your API logs to [ReadMe](http://readme.com/). Here's an overview of how the integration works:
 
-```
-composer require readme/metrics
-```
-
-## Usage
-
-`readme/metrics` is currently targeted towards codebases utilizing the [Laravel](https://laravel.com/) framework. To get up and running, you'll need to do the following things:
-
-1. Publish our config file into your `config/` directory:
+* You add the ReadMe middleware to your [Laravel](https://laravel.com/) application.
+* The middleware sends to ReadMe the response object that your Laravel application generates each time a user makes a request to your API. The entire response is sent, unless you allow or deny keys.
+* ReadMe populates Metrics with this information, such as which endpoint is being called, response code, and error messages. It also identifies the customer who called your API, using whichever keys in the middleware you call out as containing relevant customer info.
 
 ```
-php artisan vendor:publish --provider="ReadMe\ServiceProvider"
+composer require readme/metrics
 ```
 
-2. In `config/readme.php`, change `api_key` to `"<<apiKey>>"`, which is the API key for your [ReadMe project](https://dash.readme.io).
-3. In that config file, you will also see a `group_handler` that will be set to `App\Handler\ReadMe::class`. This file will exist within `app\Handler` in your application and you should change the contents of its `constructGroup` function to return data that's relevant to your codebase and users. We've provided some simple defaults but you'll likely need to change them.
-
-Once you've done all that, add `\ReadMe\Middleware::class` into your API middleware in `app/Http/Kernel.php` and API Metrics will start streaming to your ReadMe project!
-
-### `res.headers['x-documentation-url']`
-
-With the middleware loaded, all requests that funneled through it will receive a `x-documentation-url` header applied to the response. The value of this header will be the URL on ReadMe Metrics with which you can view the log for that request.
-
-Note that in order to generate this URL, an API request is made to ReadMe once a day, and cached to a local file in `$COMPOSER_HOME/cache`, to retrieve your projects `base_url`. If this request to ReadMe fails, the `x-documentation-url` header will not be added to responses.
-
-If you wish to not rely on this cache, you can opt to supply a `base_log_url` option within `config/readme.php`. This value should evaluate to the public-facing URL of your ReadMe project.
-
-### Configuration Options
-
-There are a few options you can pass in to change how the logs are sent to ReadMe. These are configured within `config/readme.php`.
+* [Integration Documentation](https://docs.readme.com/docs/sending-logs-to-readme-with-php-laravel)
```

Comment:
I don't love that this bulletpoint is by itself but I also don't have any ideas to not have it like this.

Reply:
How about this?
```suggestion
**For more information on setup, check out our [integration documentation](https://docs.readme.com/docs/sending-logs-to-readme-with-php-laravel).**
```